RadarSDK/RadarAPIClient.m

```diff
@@ -264,11 +264,18 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
     } else {
         params[@"xPlatformType"] = @"Native";
     }
+    NSMutableArray<NSString *> *fraudFailureReasons = [NSMutableArray new];
     if (@available(iOS 15.0, *)) {
         CLLocationSourceInformation *sourceInformation = location.sourceInformation;
         if (sourceInformation) {
             if (sourceInformation.isSimulatedBySoftware || sourceInformation.isProducedByAccessory) {
                 params[@"mocked"] = @(YES);
+                if (sourceInformation.isSimulatedBySoftware) {
+                    [fraudFailureReasons addObject:@"fraud_mocked_from_mock_provider"];
```

Comment:
@tderouin @lmeier This reason will also be used for `location.isFromMockProvider` on Android/Windows